influxdb_client_3/write_client/client/write/dataframe_serializer.py

```diff
@@ -160,6 +160,11 @@ def __init__(self, data_frame, point_settings, precision=DEFAULT_WRITE_PRECISION
         # null_columns has a bool value for each column holding
         # whether that column contains any null (NaN or None) values.
         null_columns = data_frame.isnull().any()
+
+        # inf_columns has a bool value for each column holding
+        # whether that column contains any Inf values.
+        inf_columns = data_frame.isin([np.inf, -np.inf]).any()
+
```

Comment:
What do you think about simply replacing `inf` values with `NaN`? For example:


```suggestion
        # Replace inf values with NaN
        data_frame.replace([np.inf, -np.inf], np.nan, inplace=True)
        null_columns = data_frame.isnull().any()

```

Given that we have our own copy of the DataFrame as shown here: https://github.com/InfluxCommunity/influxdb3-python/blob/a1d295e64db231fbac8f4c6cf4e221717b4594e3/influxdb_client_3/write_client/client/write/dataframe_serializer.py#L103, I think we can do this operation.

Reply:
But it is a shallow copy (`deep=False`), therefore `"Any changes to the data of the original will be reflected in the shallow copy (and vice versa)."`,  according to [pandas.DataFrame.copy](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.copy.html), or am I getting wrong?

Reply:
I missed mentioning 'and vice versa', so let's stick with using `null_columns`.